plasma_framework/docs/design/exit_bond_and_bounty_mechanics.md

```diff
@@ -0,0 +1,196 @@
+# Exit Bond and Bounty Mechanics
+
+This document is a description of the current implementation of exit bonds and process exit bounties which are an integral part of the exit mechanism.
+### Terminology
+
+- **Exit Initiator** The person that starts an exit by either calling `startStandardExit()` or piggybacking on an in-flight exit
+- **Challenger** The person challenges a standard exit, an in-flight exit or a piggyback
+- **Exit Processor** The person that calls `processExit()`
+- **Exit Bond** The bond put up by the **Exit Initiator**
+- **Process Exit Bounty** A portion of the **Exit Bond** that is give to the **Exit Processor** as a reward for processing the exit.
+Whenever an exit is started, the **Exit Initiator** must put up an **Exit Bond**.
+If the exit is proven to be invalid the entire **Exit Bond** is awarded to the **Challenger**
+If the exit is valid then after the challenge period it can be **processed** i.e. the funds returned to the **Exit Initiator**. A portion of the **Exit Bond**, called the **Process Exit Bounty**, is awarded to the **Exit Processor**. The remainder of the **Exit Bond** is returned to the **Exit Initiator**.
+Note that the **Exit Initiator** can also be the **Exit Processor**.
+
+## The Bond Life Cycle
+
+### For Standard Exits
+
+The **Exit Bond** is supplied by the **Exit Initiator** when starting an exit. Its main motive is to disincentivize invalid exits, but in the case of a valid exit a portion of it is used to incentivize processing the exit by rewarding the **Exit Processor**.
+The **Exit Initiator** should obtain the correct size of the Exit Bond by calling `startStandardExitBondSize()`.
+
+**_After Starting an Exit:_**
+*Bond -> from `Exit Initiator` to `Exit Game Contract`*
+
+For an invalid exit, a challenge will transfer the whole bond to the challenger. Non-exitable exits would still remain in the exit queue and is deleted when the exits are processed. For a scenario where there are mutliple invalid exits in the queue, the processor is still awarded with some gas reward for deleting non-exitable exits from storage.
```

Comment:
```suggestion
For an invalid exit, a challenge transfers the whole bond to the challenger. Non-exitable exits still remain in the exit queue and are deleted when the exits are processed. For a scenario where there are mutliple invalid exits in the queue, the processor is still awarded with some gas reward for deleting non-exitable exits from storage.
```

Reply:
`For a scenario where there are mutliple invalid exits in the queue, the processor is still awarded with some gas reward for deleting non-exitable exits from storage.`

I case the exits are challenged successfully the exit processor does not get anything right?

Reply:
https://github.com/omisego/plasma-contracts/blob/8468675915fdb202cf4656644621f7d4ebaea0da/plasma_framework/contracts/src/exits/payment/controllers/PaymentProcessStandardExit.sol#L58-L63

Reply:
yes, except only for a small gas award from EVM for deleting storage